.github/workflows/tests.yml

```diff
@@ -346,7 +346,7 @@ jobs:
         run: |
           function remove_files() {
               local extension=$1
-              find ./result_images -type f -name "*-expected*.$extension" | while read file; do
+              find ./result_images -name "*-expected*.$extension" | while read file; do
```

Comment:
`find ./result_images \( -xtype f -o -type f \) -name "*-expected*.$extension"` would more specifically target files and symlinks to files. Don't think it matters too much either way, as we aren't doing an `rm -r` later that would mistakenly take out directories.

Reply:
Ugh, never mind.  `find` on MacOS does not support the `-xtype` option.  I have reverted to avoiding any type constraints.  It's difficult to imagine that there will ever be a directory named (for example) `...-expected.png/`, so I think it's safe.